src/components/Cards/CourseCard.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react'
+import { Text, View, StyleSheet } from 'react-native'
+
+class CourseCard extends Component {
```

Comment:
Could you rewrite this as a functional component instead of a class component? Just for sake of consistency with our other components and the existing CT codebase